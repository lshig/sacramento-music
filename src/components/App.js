import React, { Fragment } from 'react';
import Charts from './Charts';
import Heading from './Heading';
import Intro from './Intro';
import SEO from './SEO';
import Table from './Table';

export default function App() {
  return (
    <Fragment>
      <SEO />
      <Heading />
      <Intro />
      <Charts />
      <Table />
    </Fragment>
  );
}
