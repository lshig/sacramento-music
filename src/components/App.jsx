import React from 'react';
import Heading from './Heading';
import Intro from './Intro';
import Chart from './Chart';
import Table from './Table';
export default function App () {
  return (
    <div>
      <Heading />
      <Intro />
      <Chart />
      <Table />
    </div>
  );
}
