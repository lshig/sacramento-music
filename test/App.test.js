import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import App from '../src/components/App';

jest.mock('highcharts', () => ({
  setOptions: jest.fn()
}));

describe('App', () => {
  describe('basic', () => {
    const wrapper = shallow(<App />);

    test('renders', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
