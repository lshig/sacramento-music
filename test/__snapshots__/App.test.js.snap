// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App basic renders 1`] = `
<Fragment>
  <SEO />
  <Heading />
  <Intro />
  <Chart />
  <Table />
</Fragment>
`;
